nazgul/mysql_model.py

```diff
@@ -432,6 +458,16 @@ def _reset_db(self):
         cur.close()
         self._db.commit()
 
+    def _db_heartbeat(self):
```

Comment:
Lets refactor this method in a `_get_cursor()` function that will return `self._db.cursor()` and if it fails, restart the connection